Lib/distutils/util.py

```diff
@@ -38,7 +38,7 @@ def get_host_platform():
     if os.name == 'nt':
         if 'amd64' in sys.version.lower():
             return 'win-amd64'
-        if 'arm32' in sys.version.lower():
+        if 'arm' in sys.version.lower():
```

Comment:
sys.version() return ARM not arm32
>>> sys.version
'3.8.0a3+ (heads/test:5145ef5d65, Apr 19 2019, 14:58:01) [MSC v.1916 32 bit (ARM)]'